db/queries.ts

```diff
@@ -248,7 +251,7 @@ export function getUsername(supabase: SupabaseClient<Database>, userId: string,
 				username = data.username;
 			}
 
-			return { username, full_name: data?.full_name };
```

Comment:
This branch is out of sync with dev (see #29). This branch should be rebased/resolved before merging changes as this sets back some of the bug fixes in previous PRs.

Reply:
I am taking a closer look at this tmr morning but this does seem problematic based on #29

Reply:
Yea the branch head is out of date and merging sets back 29.